src/utils/getCropLifecycleDuration.js

```diff
@@ -6,6 +6,12 @@ import { memoize } from './memoize'
  * @param {{ cropTimetable: Object }} crop
  * @returns {number}
  */
-export const getCropLifecycleDuration = memoize(({ cropTimetable }) =>
-  Object.values(cropTimetable).reduce((acc, value) => acc + value, 0)
-)
+export const getCropLifecycleDuration = memoize(({ cropTimetable }) => {
+  return Object.values(cropTimetable).reduce((acc, value) => {
+    if (Array.isArray(value)) {
+      return acc + value.reduce((acc2, value2) => acc2 + value2)
+    }
+
+    return acc + value
+  }, 0)
+})
```

Comment:
Let's add some test coverage for the additional logic path here: https://github.com/jeremyckahn/farmhand/blob/cfdab66a5347427e16310e8e65b0484b08a08a1b/src/utils/getCropLifecycleDuration.test.js#L7-L11

Reply:
i think now that i'm using a real crop in the test and all crops have the array GROWING phase this is naturally covered